### PR TITLE
Upload the 2024 community survey blog

### PR DESCRIPTION
**Pull request by googanhiem**
## Proposed change

Upload the 2024 community survey blog 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new blog post announcing the 2024 Home Assistant community survey, emphasizing inclusivity and user experiences.
	- Added a new entry for Annika Schulz in the people directory.

- **Bug Fixes**
	- Ensured proper formatting by adding a newline at the end of Missy Quarry's entry.

### CODE REVIEW

File: source/_posts/2024-12-16-community-survey-2024.markdown [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+date_formatted: "December 16, 2024"
+author: Annika Schulz
+comments: true
+categories: Announcements
+og_image: /images/blog/2024-12-survey/art.jpg
+---
+
+<img src='/images/blog/2024-12-survey/art.jpg' alt="2024 community survey">
+
+As Home Assistant continues to grow and evolve, so does our commitment to making it more inclusive, accessible, and aligned with the diverse needs of our community. To that end, we're launching an annual survey---and we hope you'll take part!
```
Comment (frenck) [inline source/_posts/2024-12-16-community-survey-2024.markdown]:
```suggestion
As Home Assistant continues to grow and evolve, so does our commitment to making it more inclusive, accessible, and aligned with the diverse needs of our community. To that end, we're launching an annual survey&mdash;and we hope you'll participate!
```

File: source/_posts/2024-12-16-community-survey-2024.markdown [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+- Tailor our future work to address the needs of our community.
+- Identify trends and shifts in the smart home landscape over time.
+- Ensure Home Assistant and its ecosystem reflect the values and priorities of those who use it.
+
+We understand that some of the questions we're asking touch on sensitive topics. Rest assured, your responses are completely anonymous, and all questions are optional. If you're ever uncomfortable, you can skip a question---but for the results to be recorded, you must hit the submit button at the end of the survey. This survey isn't short; we recommend setting aside around 20 minutes to complete it. [**Fill out the survey here**](https://home-assistant.typeform.com/communitysurvey#source=blog) or read more about what we're trying to learn below.
+
+~Annika
```
Comment (frenck) [inline source/_posts/2024-12-16-community-survey-2024.markdown]:
```suggestion
We understand that some of the questions we're asking touch on sensitive topics. Rest assured, your responses are completely anonymous, and all questions are optional. If you're ever uncomfortable, you can skip a question&mdash;but for the results to be recorded, you must hit the submit button at the end of the survey. This survey isn't short; we recommend setting aside around 20 minutes to complete it. [**Fill out the survey here**](https://home-assistant.typeform.com/communitysurvey#source=blog) or read more about what we're trying to learn below.

~Annika
```

File: source/_posts/2024-12-16-community-survey-2024.markdown [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+<!--more-->
+
+## What we're hoping to learn
+
+We're curious about how you began your smart home journey and what drives your decisions today in your home. From the types of devices you use to how your household interacts with your smart home. We want to learn who is using Home Assistant, not just the primary user, but also those living in your smart home. These insights allow us to understand the varied experiences and expertise within the community. For example, knowing your professional background helps us identify areas where your day-to-day knowledge intersects with smart home technologies. We are also asking about neurodiversity, which can help us understand how our community processes and interacts with information. By learning about these differences, we can make our communication and platform design more inclusive. Again, we can understand if you prefer not to share this information.
+
+One area we're trying to understand better is how you personally resonate with the values of the Open Home. We believe in privacy, sustainability, and choice---but how important are these values to you?
```
Comment (frenck) [inline source/_posts/2024-12-16-community-survey-2024.markdown]:
```suggestion
One area we're trying to understand better is how you personally resonate with the values of the Open Home. We believe in privacy, sustainability, and choice&mdash;but how important are these values to you?
```

File: source/images/blog/2024-12-survey/ art.jpg
Comment (frenck) [inline source/images/blog/2024-12-survey/ art.jpg:1]: There seems to be a space in the filename?

CleanShot 2024-12-16 at 17 18 27

Comment (googanhiem) [reply source/images/blog/2024-12-survey/ art.jpg:1]: weird, looking at my codespace it doesn't have this space. Anyways, fixed it manually.

**Review by googanhiem — COMMENTED** [review]
This is ready to go

**Review by frenck — APPROVED** [review]

**Pull request merged**